internal/attestation/measurements/measurements.go

```diff
@@ -325,3 +342,28 @@ type encodedMeasurement struct {
 	Expected string `json:"expected" yaml:"expected"`
 	WarnOnly bool   `json:"warnOnly" yaml:"warnOnly"`
 }
+
+// mYamlContent is the Content of a yaml.Node encoding of an M. It implements sort.Interface.
+// The slice is filled like {key1, value1, key2, value2, ...}.
+type mYamlContent []*yaml.Node
+
+func (c mYamlContent) Len() int {
+	return len(c) / 2
+}
+
+func (c mYamlContent) Less(i, j int) bool {
+	lhs, err := strconv.Atoi(c[2*i].Value)
+	if err != nil {
+		panic(err)
+	}
+	rhs, err := strconv.Atoi(c[2*j].Value)
+	if err != nil {
+		panic(err)
+	}
+	return lhs < rhs
+}
+
+func (c mYamlContent) Swap(i, j int) {
+	c[2*i], c[2*j] = c[2*j], c[2*i]
```

Comment:
We should repeat the comment from `mYamlContent` about the `key1, value1, key2, value2, ...` structure to explain the `2*` everywhere. 
ngl.. I was a bit confused whats going on here😆